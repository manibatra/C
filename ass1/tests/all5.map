12 11
...E.\.....
...........
.@=......=.
...........
..C......D.
...........
...........
.......B...
..A........
.....F.....
.\......./.
...........
