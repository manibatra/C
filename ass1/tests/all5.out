..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
........\/.

(side pos)>..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
..1........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
..1........
..1........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
..1........
..1........
..1........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
..1........
..1........
..1........
..1........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
..1........
..1........
..1........
..1........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

..=..../\/\
..1........
..1........
..1........
..1........
..@........
/........./
\.........\
/........./
\.........\
..@........
..1.....\/.

End of simulation.
..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
........\/.

(side pos)>..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\........1\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.......11\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\......111\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.....1111\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\....11111\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\...111111\
..@.......1
........\/1

..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\..1111111\
..@.......1
........\/1

End of simulation.
..=..../\/\
...........
...........
...........
...........
..@........
/........./
\.........\
/........./
\.........\
..@........
........\/.

(side pos)>